src/crypto/bcrypt.cr

```diff
@@ -5,6 +5,9 @@ require "./subtle"
 # Mazières, as [presented at USENIX in
 # 1999](https://www.usenix.org/legacy/events/usenix99/provos/provos_html/index.html).
 #
+# The algorithm has a maximum password length limit of 72 characters (see
+# [this answer](https://security.stackexchange.com/questions/39849/does-bcrypt-have-a-maximum-password-length#answer-39851)).
```

Comment:
It's 72 bytes, including the terminating NULL byte, which is expected by the algorithm, thus 71 bytes are available, which may be even less characters...

Sigh. The more I read about bcrypt, the more I think it's really weird (why encode the null byte? why a limit?), and we really should move on to [scrypt](https://github.com/ysbaddaden/scrypt-crystal) and [argon2](https://en.wikipedia.org/wiki/Argon2) (argon2id).

"See [this answer]" is weird. Maybe "See [this comment] on stackoverflow"?

Reply:
OK, thanks. I agree on providing scrypt, but since we started with bcrypt, people will definitely want it in the stdlib.

Reply:
I was thinking of a "password" shard, with different solutions, such as bcrypt (for compatibility), scrypt, the argon2 variants, ...

Reply:
Fixed.